terraform/helm/aptos-node/templates/validator.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "aptos-validator.fullname" . }}-validator
+  labels:
+    {{- include "aptos-validator.labels" . | nindent 4 }}
+spec:
+  selector:
+    {{- include "aptos-validator.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/name: validator
+  ports:
+  - name: validator
+    port: 6180
+  - name: fullnode
+    port: 6181
+  - name: metrics
+    port: 9101
+  {{- if .Values.validator.exposeRestApi }}
+  - name: api
+    port: 8080
+  {{- end }}
+
+---
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "aptos-validator.fullname" . }}-validator-e{{ .Values.chain.era }}
```

Comment:
Perhaps make a new PVC when `.Values.loadTestGenesis = true`? Otherwise, operators will have to delete their test PVCs when spinning up their IT1 nodes, and might cause confusion.